projects/nano-editor.org/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.raw }}.tar.gz
+
+versions:
+  - 7.2
+
+dependencies:
+  gnu.org/gettext: "*"
+  invisible-island.net/ncurses: ">=6.0"
+
+provides:
+  - bin/nano
+
+build:
+  dependencies:
+    freedesktop.org/pkg-config: "*"
+    gnu.org/automake: "*"
```

Comment:
Almost certainly should be `^2`. I doubt automake will ever release another major version but I am certain that if they do it will be radically different and break everything that assumed otherwise.

Reply:
we have only bottled 1.16.5 atm